### PR TITLE
Rename 'user management' app to 'admin settings'

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
The "user management" app has been renamed to "admin settings" because the app will contain a variety of general settings in the future.

## Related Issue

- https://jira.owncloud.com/browse/OCIS-3190

## Follow-up
- [ ] Update the web config in oCIS after a new Web RC has been created

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

File: changelog/unreleased/enhancement-rename-user-management [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Enhancement: Rename "user management" app
+
+The "user management" app has been renamed to "admin settings" because the app will contain a variety of general settings in the future.
+
+https://github.com/owncloud/web/pull/8175
```
Comment (dschmidt) [inline changelog/unreleased/enhancement-rename-user-management:5]: It's a breaking change, as it potentially needs a config adjustment by the user. 


We could also do a graceful deprecation and keep the old name working in config but pring a warning when it's used... 

Then it should also be added to the deprecations ticket

Comment (JammingBen) [reply changelog/unreleased/enhancement-rename-user-management:5]: Hmm I'm not sure actually, according to the Jira ticket, it's not a breaking change due to https://github.com/owncloud/web/pull/8175/files#diff-9521d586d9cc2dae8e0510f697d29d0fc9b164419ebc855a5bca4e216414e96e. That also prints a warning

Comment (dschmidt) [reply changelog/unreleased/enhancement-rename-user-management:5]: Oh you're right, apparently scrolled over that! Happy with that, nvm!

Comment (dschmidt) [reply changelog/unreleased/enhancement-rename-user-management:5]: Can you still add it to the deprecations list?

Comment (JammingBen) [reply changelog/unreleased/enhancement-rename-user-management:5]: Done so, thx for the hint!

**Review by fschade — APPROVED** [review]
LGTM

**Pull request merged**